paf-mvp-frontend/src/services/http.service.ts

```diff
@@ -12,74 +12,148 @@ export interface IHttpService {
    * Stringify the JS object and HTTP-POST it at the given URL
    * with headers including 'credentials: include'.
    *
+   * Return a rejected promise if the service is currently redirecting.
```

Comment:
*NIT: Return**s**. The same applies for other verbs in the comments.

Reply:
Done and applied on the file.